xAPISGProcessor.ipynb

```diff
@@ -39,321 +39,408 @@
+    "    games_started_completed=vis_games_started_completed()\n",
+    "    #Bar chart with number of times each video was seen and skipped\n",
+    "    %run vis/xAPISG-VideosSeenSkipped.ipynb\n",
+    "    videoseenskipped=vis_videos_seen_skipped()\n",
+    "    #Line chart with progress of players per time\n",
+    "    %run vis/xAPISG-PlayersProgress.ipynb\n",
+    "    playerprogress=vis_players_progress()\n",
+    "    #Bar chart with scores of players in each completable\n",
+    "    %run vis/xAPISG-CompletablesScores.ipynb\n",
+    "    completable_score=vis_completables_scores()\n",
+    "    #Bar chart with progress of players in each completable\n",
+    "    %run vis/xAPISG-CompletablesProgress.ipynb\n",
+    "    completable_progress=vis_completables_progress()\n",
+    "    #Bar chart with max and min completion time in each completable\n",
+    "    %run vis/xAPISG-CompletablesTimes.ipynb\n",
+    "    completable_time=vis_completables_times()\n",
+    "    completabletab=VBox([completable_score, completable_progress, completable_time])\n",
+    "    #Bar chart with number of correct and incorrect answers per player\n",
+    "    %run vis/xAPISG-CorrectIncorrectPerPlayer.ipynb\n",
+    "    resaltperplayer=vis_correct_incorrect_player()\n",
+    "    #Bar chart with number of correct and incorrect answers per question (alternative)\n",
+    "    %run vis/xAPISG-CorrectIncorrectPerQuestion.ipynb\n",
+    "    resaltperalternative=vis_correct_incorrect_question()\n",
+    "    #Bar chart per question, with number of times each alternative has been selected\n",
+    "    %run vis/xAPISG-AlternativesSelectedQuestion.ipynb\n",
+    "    resalt=vis_alternatives_selected_question()\n",
+    "    alternativetab=VBox([resaltperplayer, resaltperalternative, resalt])\n",
+    "    tabs=widgets.Tab(children=[games_started_completed, videoseenskipped, playerprogress, completabletab,alternativetab])\n",
+    "    tabs.set_title(0,\"Games started and Completed\")\n",
+    "    tabs.set_title(1,\"Videos seen and skipped\")\n",
+    "    tabs.set_title(2,\"Progress\")\n",
+    "    tabs.set_title(3,\"Completable\")\n",
+    "    tabs.set_title(4,\"Alternative\")\n",
+    "    app=VBox([checkboxesPlayersSelected, tabs])\n",
+    "    return app"
+   ],
+   "metadata": {
+    "collapsed": false,
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   }
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
-    "**Bar chart per question, with number of times each alternative has been selected**"
-   ]
+    "**File explorer** to find the file you want to analyse with **Statement Processor** to generate all data and visualisations"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
   },
   {
    "cell_type": "code",
-   "execution_count": 12,
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "%run ProcessxAPISGStatement.ipynb # notebook to process an xAPI-SG statement\n",
```

Comment:
Revise if some parts of this code (and maybe also from previous cells) could be hidden in other notebooks, as we want to keep the main notebook as some higher-level user-friendly dashboard.